src/components/TradesWidget/TradeRow.tsx

```diff
@@ -76,6 +78,61 @@ const TypePill = styled.span<{
   text-transform: uppercase;
 `
 
+interface MarketAndPricesParams extends Pick<Trade, 'buyToken' | 'sellToken' | 'limitPrice' | 'fillPrice'> {
+  isPriceInverted: boolean
+}
+
+interface MarketAndPrices {
+  side: string
+  quoteTokenLabel: React.ReactNode
+  sellTokenLabel: React.ReactNode
+  buyTokenLabel: React.ReactNode
```

Comment:
can we stick to a consistent `buyToken` OR `receiveToken` it's mixed in several places. I believe `receiveToken` is the standard